packages/manager/apps/hub/src/dashboard/dashboard.html

```diff
@@ -9,15 +9,20 @@ <h2 data-translate="manager_hub_dashboard_overview"></h2>
 <div class="row">
     <div class="col-md-8">
         <hub-payment-status
-            data-services="$ctrl.billingServices.data"
-            data-tracking-prefix="{{ $ctrl.trackingPrefix }}"
-            data-total-count="$ctrl.billingServices.count"
+            services="$ctrl.billingServices.data"
+            tracking-prefix="{{ $ctrl.trackingPrefix }}"
+            total-count="$ctrl.billingServices.count"
+            refresh="$ctrl.refreshBillingServices()"
```

Comment:
prefix with data- for all custom attributes

Reply:
@radireddy Not mandatory as those are custom components